package.json

```diff
@@ -383,6 +384,28 @@
                     "default": "TargetLanguageCode",
                     "description": "Specifies which target language to use by default in the XLIFF Sync snippets.",
                     "scope": "resource"
+                },
+                "xliffSync.defaultLanguages": {
```

Comment:
I wanted to leave this empty because `createNewTargetFiles` also uses this setting and when empty asks for the languages to use as it would do normally.

Reply:
But you mean that's also when explicitly defining the default to be empty?

Reply:
Already set the default to empty 👍